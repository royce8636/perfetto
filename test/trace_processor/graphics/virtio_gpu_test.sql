SELECT
  ts,
  dur,
  name
FROM
  slice
ORDER BY ts;
