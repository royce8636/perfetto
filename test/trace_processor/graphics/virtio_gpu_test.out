"ts","dur","name"
1345090723759,1180312,"SUBMIT_3D"
1345090746311,1167135,"CTX_DETACH_RESOURCE"
